library/quality/quality.gradle

```diff
@@ -69,8 +69,7 @@ task pmd(type: Pmd) {
 
 android {
     lintOptions {
-        disable 'InvalidPackage',
-                'NewerVersionAvailable' // Comment this line out to do a quick check for library version updates.
+        disable 'InvalidPackage', 'RestrictedApi' // TODO remove RestrictedApi
```

Comment:
I think yes.  Given that we don't need alpha3 for anything in particular it seems like a good candidate for skipping.